spec/ref_arch_setup/bolt_helper_spec.rb

```diff
@@ -4,6 +4,7 @@
   let(:nodes)        { "local://localhost" }
   let(:dir)          { "/tmp/ref_arch_setup" }
   let(:cmd)          { "echo foo" }
+  let(:task)         { "test_task" }
```

Comment:
It probably doesn't matter, but we should probably use something like foo::bar since that is what we will be passing.  Just in case at some point some text processing happened... then we can be sure it doesn't get messed up.